core/src/main/scala/com/snowplowanalytics/snowplow/eventgen/protocol/event/UnstructEvent.scala

```diff
@@ -57,7 +57,7 @@ object UnstructEvent {
         "elementId" -> elementId,
         "nodeName"  -> nodeName,
         "type"      -> `type`.getOrElse(""),
-        "value"     -> value.orNull
+        "value"     -> value.getOrElse("")
```

Comment:
I think we do want a possible `null` here, because the schema [calls for it](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0#L47) and `value` is a [required property](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0#L52). 

So I think the better change might be:

```
Map(
  "formId"    -> Some(formId),
  "elementId" -> Some(elementId),
  "nodeName"  -> Some(nodeName),
  "type"      -> Some(`type`.getOrElse("")), // we do want an empty String here, 
  "value"     -> value
).filterNot { case (_, v) => v == Some("") }.asJson
```

WDYT?